Code/Tools/FBuild/FBuildTest/Tests/TestDistributed.cpp

```diff
@@ -373,6 +375,35 @@ void TestDistributed::WarningsAreCorrectlyReported_Clang() const
     TEST_ASSERT( GetRecordedOutput().Find( "warning: unused variable 'x' [-Wunused-variable]" ) );
 }
 
+// WarningsAreCorrectlyReported_Clang
+//------------------------------------------------------------------------------
+void TestDistributed::WarningsAreCorrectlyReported_ClangTidy() const
+{
+    FBuildTestOptions options;
+    options.m_ConfigFile = "Tools/FBuild/FBuildTest/Data/TestDistributed/WarningsAreCorrectlyReported/fbuild.bff";
+    options.m_AllowDistributed = true;
+    options.m_NumWorkerThreads = 1;
+    options.m_NoLocalConsumptionOfRemoteJobs = true; // ensure all jobs happen on the remote worker
+    options.m_AllowLocalRace = false;
+    options.m_ForceCleanBuild = true;
```

Comment:
ForceCleanBuild not needed - this test doesn't save build state so is always clean.